### PR TITLE
uart16550: fix files registration in dev directory

### PR DESCRIPTION
**Pull request by HBuczynski**
## Description

The server didn't wait on root. For this reason files have not been registered in `/dev` directory.

## Motivation and Context

PP-117

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [ ] Tested by hand on: (zynq7000-pilot, ia32-generic).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing linter checks and tests passed.
- [ ] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [ ] I will merge this PR by myself when appropriate.

### CODE REVIEW

File: tty/uart16550/uart16550.c [hunk 1]
```diff
@@ -242,6 +243,31 @@ static void poolthr(void *arg)
...
 
 
+static void _uart_mkDev(uint32_t port)
+{
+	oid_t dir;
+	char path[12];
+	unsigned int i;
+
+	while (lookup("/", NULL, &dir) < 0) {
+		usleep(10000);
```
Comment (agkaminski) [inline tty/uart16550/uart16550.c:253]: Missing `#include <unistd.h>`

**Review by agkaminski — DISMISSED** [review]

File: tty/uart16550/uart16550.c [hunk 2]
```diff
@@ -242,6 +244,31 @@ static void poolthr(void *arg)
 }
 
 
+static void _uart_mkDev(uint32_t port)
+{
+	oid_t dir;
+	char path[12];
+	unsigned int i;
+
+	while (lookup("/", NULL, &dir) < 0) {
```
Comment (nalajcie) [inline tty/uart16550/uart16550.c:253]: not sure You need `/`, `create_dev` internally waits for `devfs`, You should create devices there and then bind to root when available (see DTR project)

Comment (HBuczynski) [reply tty/uart16550/uart16550.c]: Thanks, it works without waiting on `/`, removed `while` loop.
Yeah, in `phoenix-pilot-project` there is the same binding as in `DTR` project.

File: tty/uart16550/uart16550.c [hunk 3]
```diff
@@ -242,6 +244,27 @@ static void poolthr(void *arg)
 }
 
 
+static void _uart_mkDev(uint32_t port)
+{
+	oid_t dir;
```
Comment (agkaminski) [inline tty/uart16550/uart16550.c:249]:
```suggestion
```

**Pull request merged**